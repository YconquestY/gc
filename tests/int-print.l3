;; In Emacs, open this file in -*- Scheme -*- mode

(int-print 2018)
(@byte-write 10)                        ;newline
(int-print -2018)
(@byte-write 10)                        ;newline
(int-print 0)
(@byte-write 10)                        ;newline
(int-print 1073741823)
(@byte-write 10)                        ;newline
(int-print -1073741824)
(@byte-write 10)                        ;newline
