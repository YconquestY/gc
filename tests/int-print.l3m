../library/integers.l3m
int-print.l3
