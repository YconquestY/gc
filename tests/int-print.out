2018
-2018
0
1073741823
-1073741824
